atf_core/src/atf_core/recorder.py

```diff
@@ -127,6 +150,12 @@ def __init__(self, test):
 
         rospy.loginfo("ATF recorder: started!")
 
+    def check_for_timeout(self, start_time, timeout):
```

Comment:
this function should accept an optional `msg` field - which can be used for additional info (such as last diagnostics)

Reply:
done in https://github.com/floweisshardt/atf/pull/186

Reply:
cherry-picked relevant changes from #186